src/layouts/components/Homepage/HeroDropdownSection.tsx

```diff
@@ -64,7 +60,8 @@ export const HeroDropdownSection = ({
         <Text mt="0.5rem" textStyle="body-2" textColor="base.content.medium">
           Drag and drop dropdown options to rearrange them
         </Text>
-        <Editable.Accordion>
+        {/* TODO: Add `displayHandler` */}
+        <Editable.Accordion onChange={() => onDisplay("dropdownelem")}>
```

Comment:
Is `index` not required for `onDisplay` (since we're passing `displayHandler`)?

Reply:
we never actually pass `displayHandler` HAHAHAHHA

Reply:
it's only required for `section`! for `dropdown/highlight`, it automatically scrolls to index 0 (ie, the hero section)!